src/SignUpIn/SignUpInController.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import createReactClass from 'create-react-class';
+import './SignUpIn.css';
+import UserTypeController from './UserTypeController';
+import SignUpIn from './SignUpIn';
+import SignIn from './SignIn';
+
```

Comment:
missing a lot of semicolons in this file

Reply:
There can't be semicolons when settings the state. Fields are separated by commas (i.e. this.setState({a:1, b:2}) and a single field has no punctuation (i.e. this.setState({a:1}) ).